docs/mkdocs.yml

```diff
@@ -0,0 +1,67 @@
+# Meta parameters for mkdocs
+docs_dir: src/
+
+# Project information
+site_name: "Simpleflow documentation"
+copyright: "Copyright &copy; 2017 - Botify"
```

Comment:
No I didn't really have opinions on the subject ^^
Just seemed weird since this is opensource and we don't actually have a copyright on it. No change needed :)